### PR TITLE
Making this a Kotlin library rather than an Android library.

### PR DESCRIPTION
**Pull request by wellingtoncosta**

### CODE REVIEW

**Comment by wellingtoncosta** [comment]
I think you could publish this on JCenter instead of Jitpack. Ok, I know that Jitpack publishing process is more easy rather than JCenter or Maven Central, but, yeah, it's just a suggestion.

File: README.md [hunk 1]
```diff
@@ -2,47 +2,51 @@
 
 Rest Error Interceptor
 ===================
-This will intercept the codes responses and throw the proper exception. </br>
+
+This interceptor provides a easy way to handle the most common REST HTTP status code as exceptions.
```
Comment (jeancsanchez) [inline README.md]: Just fix that :)

**Comment by wellingtoncosta** [comment]
Fixed, @jeancsanchez! :D

**Comment by jeancsanchez** [comment]
@WellingtonCosta Thanks for creating the unit tests! \õ/

**Pull request merged**